recipes-iec61850/iec61850-open-server/iec61850-open-server.bb

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2021, Alliander (http://www.alliander.com)
+# SPDX-License-Identifier: Apache-2.0
+
+LICENSE = "GPL-3.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=d32239bcb673463ab874e80d47fae504"
+
+SRC_URI = "git://github.com/robidev/iec61850_open_server.git;protocol=https;branch=master"
```

Comment:
do you plan to keep this repo in your own GH account ? Shall you transfer it into a corporate account: Seapath or Alliander.

Reply:
I will discuss this internally

Reply:
I have discussed it, and Alliander is not able to adopt the code. So that leaves my own private repo, or donating it to seapath, and include it in that repo. Perhaps an item for the coming TSC meeting